app/views/games/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="text-center">
   <br>
-  <%= @game.name %>
+  <!-- <%= @game.name %> -->

```

Comment:
This makes game name become game status for some reason. Since you type in a game name when you make a new game, should we want this displayed? (also not critical and can be fixed later)
![gamenameisopen](https://user-images.githubusercontent.com/22229082/28446604-c93e6642-6d99-11e7-81ef-01cea19a3e7e.png)